### PR TITLE
Fix typos in russian words

### PR DESCRIPTION
**Pull request by Ivanio1**

### CODE REVIEW

**Comment by Ivanio1** [comment]
@ryukzak

File: src/course_bot/general.clj [hunk 1]
```diff
@@ -24,6 +24,7 @@
  {:en
   {:general
    {:who-am-i-3 "Name: %s; Group: %s; Telegram ID: %s"
+    :who-am-i-4 "Имя: %s; Группа: %s; Telegram ID: %s; Группа лаб.1: %s"
```
Comment (ryukzak) [inline src/course_bot/general.clj:27]: Looks like a Russian text in English section.

**Review by ryukzak — CHANGES_REQUESTED** [review]
Thank you!

- In the next time better to tag me via `reviewers` in the top right side of the screen.
- Please, tag related issue in the description
- Test should be fixed.

File: src/course_bot/general.clj (hunk 1 above)
Comment (Ivanio1) [reply src/course_bot/general.clj]: It's new commit. I am trying to improve whoami command. Now I have reset this commit, so you can merge my branch with fixing typos in russian words commit.

**Comment by Ivanio1** [comment]
@ryukzak

**Review by ryukzak — APPROVED** [review]

**Pull request merged**